shogun/multiclass/MulticlassLogisticRegression.cpp

```diff
@@ -42,9 +42,9 @@ void CMulticlassLogisticRegression::init_defaults()
 
 void CMulticlassLogisticRegression::register_parameters()
 {
-	SG_ADD(&m_z, "m_z", "regularization constant",MS_AVAILABLE);
-	SG_ADD(&m_epsilon, "m_epsilon", "tolerance epsilon",MS_NOT_AVAILABLE);
-	SG_ADD(&m_max_iter, "m_max_iter", "max number of iterations",MS_NOT_AVAILABLE);
+	SG_ADD(&m_z, "m_z", "regularization constant", ParameterProperties::HYPER);
+	SG_ADD(&m_epsilon, "m_epsilon", "tolerance epsilon");
+	SG_ADD(&m_max_iter, "m_max_iter", "max number of iterations");
```

Comment:
we should introduce an attribute for parameters like this one .... these are hyper-parameters, but not in the classical sense (as in that we want to tune them). But still free parameters ...
@vigsterkr @lisitsyn ?